packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -19,16 +19,27 @@ const hooks = require('./hooks')
 const instrumentations = require('./instrumentations')
 const names = Object.keys(hooks)
 const pathSepExpr = new RegExp(`\\${path.sep}`, 'g')
+
+const convertInstrumentationName = (name) => {
+  return name.replaceAll('@', '').replaceAll('-', '').replaceAll('/', '').replaceAll('_', '').toLowerCase()
```

Comment:
```suggestion
  return name.replaceAll(/[-@_/]/g, '').toLowerCase()
```

More important though: we have a different handling in our plugin manager. I believe we should reuse the same regular expression, otherwise we might end up detecting things slightly different over time.